test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -344,13 +359,18 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
     await session.patch('index.module.css', `.button {`)
     expect(await session.hasRedbox(true)).toBe(true)
     const source = await session.getRedboxSource()
-    expect(source).toMatch('./index.module.css:1:1')
-    expect(source).toMatch('Syntax error: ')
-    expect(source).toMatch('Unclosed block')
+    expect(source).toMatch(
+      IS_TURBOPACK ? './index.module.css:1:8' : './index.module.css:1:1'
+    )
+    if (!IS_TURBOPACK) {
+      expect(source).toMatch('Syntax error: ')
+      expect(source).toMatch('Unclosed block')
+    }
     expect(source).toMatch('> 1 | .button {')
-    expect(source).toMatch('    | ^')
+    expect(source).toMatch(IS_TURBOPACK ? '    |         ^' : '    | ^')
```

Comment:
kinda think it would be cleaner to have just one big if else block, but doesn't matter